content/history/org.inc

```diff
@@ -381,11 +516,15 @@ function graphtrends($entrytypes, $firstyear, $lastyear) {
     </div>
     <div class="tab-pane fade" id="tab-trends" role="tabpanel" aria-labelledby="trends-tab">
       <h2>Trends</h2>
-      <?php graphtrends(array("1A","1B","1C"),"2001","2016");?>
-      <?php graphtrends(array("2A","2B"),"2001","2016");?>
+      <?php // LOL this date range is something everyone will forget to update in the future...
+            graphtrends(array("ma","mb","mc"),"2001","2022");
+            graphtrends(array("wa","wb","wc"),"2001","2022");
+      ?>
```

Comment:
Ahahaha maybe we should make a raceday playbook. I'll add a note to the Primer doc!

Reply:
Yeah.

I was sort of annoyed at the graphing stuff in general when I got here.  I suppose it is possible to just make it "show the last 20 years, always" just as well if you prefer.

Reply:
This was straightforward, done.

In the future, we probably want to hide the trends tab if it just renders blank graphs, but that's not technically a regression :)

Reply:
Amazing, thank you!